data/tariffplans/cache_replications/dispatcher_engine/DispatcherProfiles.csv

```diff
@@ -1,3 +1,3 @@
 #Tenant,ID,Subsystems,FilterIDs,ActivationInterval,Strategy,StrategyParameters,ConnID,ConnFilterIDs,ConnWeight,ConnBlocker,ConnParameters,Weight
 cgrates.org,Engine1,*any,,,*weight,,Engine1,,20,false,,10
-cgrates.org,Engine2,*chargers,*string:~*req.EventName:TestLoad,,*load,,Engine1,,20,false,,20
\ No newline at end of file
+cgrates.org,Engine2,*chargers,*string:~*req.EventName:TestLoad,,*weight,,Engine1,,20,false,*ratio:1,20
```

Comment:
*ratio should go into StrategyParameters instead of Conn ones. Same as for Load strategy

Reply:
Will need however to review this together.